node/pkg/common/scissors.go

```diff
@@ -20,6 +20,7 @@ var (
 // Start a go routine with recovering from any panic by sending an error to a error channel
 func RunWithScissors(ctx context.Context, errC chan error, name string, runnable supervisor.Runnable) {
 	go func() {
+		ScissorsErrors.WithLabelValues("scissors", name).Add(0)
```

Comment:
@hendrikhofstadt do you know if it'd be better to do this one line up?

Reply:
Yeah I'd do them in the function (aka one line up), but outside of the goroutine. There's no inherent value and it just adds more code to the goroutine.

Reply:
Agree